src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -195,16 +223,21 @@ struct TimeDependentMapOptions {
    * only be an identity map.
    */
   MapType<Frame::Grid, Frame::Inertial> grid_to_inertial_map(
-      bool include_distorted_map) const;
+      bool include_distorted_map, bool use_rigid_translation) const;
 
   inline static const std::string size_name{"Size"};
   inline static const std::string shape_name{"Shape"};
+  inline static const std::string translation_name{"Translation"};
 
  private:
   double initial_time_{std::numeric_limits<double>::signaling_NaN()};
   size_t initial_l_max_{};
   ShapeMap shape_map_{};
+  TranslationMap rigid_translation_map_{};
+  TranslationMap translation_map_{};
```

Comment:
Use `linear_falloff_translation_map_` instead of `translation_map_` so it's clear. Or just make it a `std::pair` of translation maps.